### PR TITLE
Image build

### PR DESCRIPTION
**Pull request by CyanID401**
WHAT
Dockerfile for the project

WHY
In order to build an image we need the Dockerfile, requirements.txt is also needed, and a change to the app's port had to be made.

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -11,4 +11,4 @@ def hello_world():
 
 
 if __name__ == "__main__":
-    app.run(port=os.environ.get("PORT", 3000), host="0.0.0.0")
+    app.run(port=os.environ.get("PORT", 5000), host="0.0.0.0")
```
Comment (vutoff) [inline app/app.py:14]: No need to change the code. You can either pass `-e PORT=5000` to the `docker run` command, or simply map it like `docker run -p 5000:3000 image_name:tag`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+USER notroot
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY app .
```
Comment (vutoff) [inline Dockerfile:16]: Here you need to change the ownership of the copied files:
```suggestion
COPY --chown=notroot:notroot app .
```

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+RUN apt-get update && \
+    apt-get install python3 python3-pip -y && \
+    groupadd -g 1234 notroot && \
+    useradd -m -u 1234 -g notroot notroot
+
+USER notroot
+
+WORKDIR /app
+
+COPY requirements.txt .
```
Comment (vutoff) [inline Dockerfile:12]: Here you need to change the ownership of the copied files:
```suggestion
COPY --chown=notroot:notroot requirements.txt .
```

**Comment by CyanID401** [comment]
@vutoff I made the changes, could you check?

**Review by vutoff — APPROVED** [review]

**Pull request closed without merging**